rds/variables.tf

```diff
@@ -60,6 +66,28 @@ variable "engine_version" {
   default     = "13.12"
 }
 
+variable "lambda_memory_sizes" {
+  description = "Configurable map of memory sizes for lambdas"
+  type        = map(number)
+  default     = {}
+}
+
+variable "lambda_timeouts" {
+  description = "Configurable map of timeouts for lambdas"
+  type        = map(number)
+  default     = {}
+}
+
+variable "max_capacity" {
+  type    = number
+  default = 4
+}
+
+variable "min_capacity" {
```

Comment:
I believe the very minimum is .5.  Should we default to that for initial deployments?

Reply:
I was investigating that this morning, but was running into Knex timeout issues during the deployment. I think safest for now is to stick with the cumulus default of 2.